src/cc/arduino/plugins/wifi101/flashers/cli/CLIFlasher.java

```diff
@@ -83,13 +78,12 @@ public void testConnection(String port) throws Exception {
 	}
 
 	@Override
-	public void updateFirmware(String port, WINC1500Firmware fw) throws Exception {
+	public void updateFirmware(String port) throws Exception {
 		List<String> cmd = new ArrayList<>();
 		cmd.add(executable.getAbsolutePath());
 		cmd.add("-port");
 		cmd.add(port);
 		cmd.add("-firmware");
-		cmd.add(fw.getFile().getAbsolutePath());
```

Comment:
Did you test if CLIFlasher still works? removing this line makes me think the FW filename could be missing from the commandline

Reply:
OK i'll try immediatly and add if required

Reply:
@facchinm @cmaglie no longer supported i remove from the updater if ok for you?

Reply:
IIRC the CLIFlahser has been replaced by the Java uploader, so for me it's ok to remove.